chatkit-core/src/main/kotlin/com/pusher/chatkit/users/UserService.kt

```diff
@@ -24,9 +24,11 @@ internal class UserService(
         val users = userIds.map { id -> getLocalUser(id).orElse { id } }
         val missingUserIds = Result.failuresOf(users).map { it }
         val localUsers = Result.successesOf(users)
+        val missingUserIdsQs = missingUserIds.map { userId -> "id={$userId}"}.joinToString(separator = "&")
```

Comment:
Do we need to escape the `userId` here?

Reply:
It wasn't being done previously, so I assumed the http library would take care of it..will look into it.